Automattic/jetpack
# [Boost] Fix Critical CSS error display, combine classes

Companion PR: https://github.com/Automattic/boost-cloud/pull/228

When generating Cloud CSS failed with a showstopper, they were sometimes not displayed in the UI correctly. Additionally, individual errors were not shown; they were being hidden in the UI.

This PR fixes those issues, and cleans up duplicate code.

## Proposed changes:
- Fixes some display bugs with Cloud CSS. When failures occurred, they were not being displayed properly.
- Simplifies our codebase at the same time, removing the separate handling for Cloud CSS and Critical CSS status objects.

### Other information:

## Jetpack product discussion
n/a

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:
* Update your local Cloud CSS service with https://github.com/Automattic/boost-cloud/pull/228
* Test Generating Critical CSS locally - both with and without errors, ensure it works.
* Test Generating Cloud CSS - both with and without errors.
